toponetx/transform/graph_to_simplicial_complex.py

```diff
@@ -60,3 +62,40 @@ def graph_2_clique_complex(G, max_dim=None):
 
     lst = filter(lambda face: len(face) <= max_dim, nx.enumerate_all_cliques(G))
     return SimplicialComplex(list(lst))
+
+
+def weighted_graph_to_Vietoris_Rips_complex(G, r, max_dim=None):
+    """Get the Vietoris-Rips complex of radius r of a weighted undirected graph. The Vietoris-Rips complex of radius
```

Comment:
Docstring one-linear should fit on one line. Then the description comes.

See anatomy of a docstring:
https://pyt-team.github.io/toponetx/contributing/index.html#docstring-examples